src/TextField/TextField.js

```diff
@@ -103,7 +114,7 @@ export default class TextField extends Component {
         {...other}
       >
         {label && (
```

Comment:
Same `? : null` comment. I would only use that pattern if `label` is guaranteed to be a boolean.